### DISCUSSION TITLE
Welcome to SPECFEM!

### DISCUSSION THREAD

**Discussion by danielpeter** [SPECFEM/specfem3d#1547] [General]
This is the landing forum for the SPECFEM codes (1D, 2D, 3D, 3D_GLOBE). All Users are welcome to open discussions, ask questions, suggest features or show off their work!

**Comment by carltape** [comment]
Thanks, Daniel! This discussion also landed in my email (which is what I want). I'll check on the settings and then send an email to the specfem email list to let people know where to watch/follow.

**Reply by ljhwang** [reply]
Discussions have also been pointed to from the CIG forum and geodynamics.org landing pages.

**Comment by carltape** [comment]
All: check out the specfem landing page at https://specfem.org/
Remind people to "watch" [this repo](https://github.com/SPECFEM/specfem3d), then set "All Activity" in order to see the specfem discussions (like this one).

**Reply by carltape** [reply]
We will establish some complete instructions for how a new user (to github) can set things up to receive email notifications of specfem discussions (if they want to).
